### ISSUE TITLE
VK_STRUCTURE_TYPE_SWAPCHAIN_CREATE_INFO_KHR missing in vulkan binding.

### ISSUE THREAD

**Issue by abisxir** [nimgl/nimgl#66]
Hi,

VK_STRUCTURE_TYPE_SWAPCHAIN_CREATE_INFO_KHR is missing vulkan binding, not a serious bug but vulkan validation 
layer complains when it is not provided to create swapchain.

**Comment by oakes** [comment]
Yeah there are a lot of enums that need to be added. In the meantime, you can do `cast[VkStructureType](1000001000)` just like in the [vulkan example](https://github.com/nimgl/nimgl/blob/master/examples/tvulkan.nim). That number comes from [vulkan.h](https://github.com/ARM-software/vulkan-sdk/blob/8f892c4edb4b440349f18ca0961387cb1d50621f/include/vulkan/vulkan.h#L204) so you can find other missing constants the same way.

**Comment by abisxir** [comment]
Thanks for quick reply, I did the that trick actually.
